portfolio/src/main/webapp/script.js

```diff
@@ -42,7 +42,10 @@ function highlightTab() {
 function fetchComments() {
   // get num of comments to query
   const selectBar = document.getElementById("numComments");
-  const maxComments = selectBar.options[selectBar.selectedIndex].value;
+  let maxComments = "All";
```

Comment:
IMO: A better API would be to simply return all comments when numComments is not specified. To retrieve all comments you would fetch('/comments') instead of fetch('/comments?numComments=All'). This would make the default/most likely call simpler.